demos/projects/ESPRESSIF/az-nvs-cert-bundle/main/az_nvs_cert_bundle_main.c

```diff
@@ -96,183 +97,216 @@ static uint8_t trust_bundle[] =
     "AHFiRSdLOkKEW39lt2c0Ui2cFmuqqNh7o0JMcccMyj6D5KbvtwEwXlGjefVwaaZB\r\n"
     "RA+GsCyRxj3qrg+E\r\n"
     "-----END CERTIFICATE-----\r\n";
-static uint32_t trust_bundle_size = sizeof( trust_bundle ) - 1;
-static uint32_t trust_bundle_version = 1;
+static uint32_t ulTrustBundleSize = sizeof( ucTrustBundle ) - 1;
+static uint8_t ucTrustBundleVersion[] = "1.0";
```

Comment:
Shouldn't this be a number: the version of the bundle should always be increasing. If the same bundle is sent, it should be ignored.

Reply:
True. It was done this way to allow for minor versions. I'm open to change, but we should make sure to change all three areas of code (sample, nvs write, and AZ function).

Reply:
The reason I'm asking is because the specification _requires_ validation that the trustbundle version is _always_ greater than the one installed.
I would assume the check is a lot easier with integers than strings.